dcos/util.py

```diff
@@ -610,3 +616,18 @@ def normalize_marathon_id_path(id_path):
 
 
 logger = get_logger(__name__)
+
+
+def hash_file(filename):
+    """Calculates the sha256 of a file
```

Comment:
`dcos.subcommand` has a `_hashfile` method for sha256. Looks like this is md5, we should name the function as such